modules/db/src/main/scala/gem/dao/UserDao.scala

```diff
@@ -33,6 +33,7 @@ object UserDao {
              g.program_role
         FROM gem_user u LEFT OUTER JOIN gem_user_program g ON g.user_id = u.id
         WHERE id = $id
+        AND "md5" = md5($pass)
```

Comment:
I'd like to avoid storing passwords, instead giving that business to ITS. We may want to have local password for development and eventually for some sort of admin user

In any case md5 hast of the password is likely too weak in this day. May I suggest something like `hmac` with  a key of at least 256 bits. I don't know how to do that in postgres though